recipes/spleeter-gpu/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [osx or py<36 or py>37]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
```

Can this be made noarch and have this included for all versions? It looks like it is available for 2.7, 3.6, 3.7 and 3.8.
```suggestion
    - importlib_resources
```

---

Nope it can't. First of all Spleeter library does only support python 3.6 and 3.7. Plus the tensorflow-gpu dependency is not supported for OSX platforms.